tests/mockers.py

```diff
@@ -117,6 +117,10 @@ def add_git_config_key(self, key: str, value: str) -> "GitRepositorySandbox":
         self.execute(f'git config {key} {value}')
         return self
 
+    def write_to_file(self, file_name: str, file_content: str):
```

Comment:
Nit: IIRC pytest has even that piece ready... I remember that Alex Burchenko used it somewhere (although maybe a false memory)

Reply:
I can't find it :(